web/packages/teleport/src/Bots/Details/Instance.tsx

```diff
@@ -46,56 +46,59 @@ export function Instance(props: {
 }) {
   const { id, version, hostname, activeAt, method, os } = props;
 
+  const hasHeartbeatData = !!version || !!hostname || !!method || !!os;
```

Comment:
is this b/c of backwards compat? if so, should we say they need to upgrade? (sorry if im way off)